<!-- -->
<launch>
    <node pkg="ga_points_downsampler" type="pcl_points_downsampler" name="pcl_points_downsampler" output="screen">
        <param name="is_downsample" value="true" />
        <param name="is_pub_pc" value="false" />
        <param name="is_use_gpu" value="false" />
        <param name="downsample_res" value="1.0" />
    </node>

</launch>